### DISCUSSION TITLE
understanding setup questions - VTT instructions are nebulous to me

### DISCUSSION THREAD

**Discussion by ClaireCJS** [microsoft/cascadia-code#778] [Q&A]
The build instructions at [https://github.com/microsoft/cascadia-code/wiki/Modifying-Cascadia-Code#building-the-font-from-source](https://github.com/microsoft/cascadia-code/wiki/Modifying-Cascadia-Code#building-the-font-from-source) have some nebulous entries in them that I think need clarifying for people not already in the weeds to understand.

For instance, I'm sure this seemed like a very simple instruction to whoever wrote it:

```
Open CascadiaCode.ttf in VTT. Run Tools > Compile > Everything for all glyphs
```

But after running the github command to clone VTT:

1. I have no idea where VTT is
2. I have no idea how to "open" something with "VTT".
3. A search for a folder named "VTT" on my computer yields no results.
4. There absolutely is not a vtt.exe anywhere on my machine

A little clarification would go a long way.

All I wanted to do is remove the "***" ligature and I'm about 2 hrs in and I thought it would be a very simple task — changing "yOffset" from "300" to "0".... but the build instructions are not sufficient for me to complete the task after changing this value. I had other nebulous errors that I put into a separate thread for support.  But a quick summary of my other error is:

```
TypeError: FeatureLibError.__init__() missing 1 required positional argument: 'location'
```

Yea? Argument where? In what file? Where is the argument missing from?  What were the other arguments (which might allow me to find the missing ones)?

The error message really needs to be .. informative.  Pointing to a file in my python's \lib\ folder is definitely not the source of the error. The source of the error is definitely me, yet.... How am I supposed to find it if the only message being thrown is basically "an error happened in a python library"?

**Comment by aaronbell** [comment]
As I mentioned on your other thread, this is a FontTools issued error, so I’d suggest taking it up with them to change the output.

As for VTT, you man have missed the link further up in the article you linked to the [homepage for Microsoft VTT](https://learn.microsoft.com/en-us/typography/tools/vtt/). As far as I know, it is not something that can be downloaded from github.
